intake_cmip/cmip5.py

```diff
@@ -43,6 +34,53 @@ def __init__(
         database : string or file handle
              File path or object for cmip5 database. For users with access to
              NCAR's glade file system, this argument can be set to 'glade'.
+        """
+
+        # store important kwargs
+        self.database = self._read_database(database)
+        self.urlpath = ""
+        self.query = {}
+        self.query_results = None
+        self._ds = None
+        super(CMIP5DataSource, self).__init__(metadata=metadata)
+
+    def _read_database(self, database):
+        if database == "glade":
+            database = glade_cmip5_db
+        if os.path.exists(database):
```

Comment:
Does this have to be a local file, or can we accept remote things like elsewhere in intake?

Reply:
It doesn't need to be a local file. I just opted for a local file for prototyping purposes for the time being. As this matures, it makes sense to support remote databases/files.